android/src/main/java/com/guichaguri/trackplayer/module/MusicModule.java

```diff
@@ -70,6 +71,11 @@ public void onServiceConnected(ComponentName name, IBinder service) {
         binder = (MusicBinder)service;
         connecting = false;
 
+        // Reapply options that user set before with updateOptions
+        if (this.options != null) {
+          binder.updateOptions(this.options);
```

Comment:
Can you indent this line with 4 spaces instead of 2?

Reply:
Sure, fixed